src/main/resources/org/jenkinsci/plugins/GithubLogoutAction/index.jelly

```diff
@@ -0,0 +1,42 @@
+<!--
+ The MIT License
+ 
+ Copyright (c) 2016 CloudBees, Inc., James Nord
+ 
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+ 
+ The above copyright notice and this permission notice shall be included in
+ all copies or substantial portions of the Software.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ THE SOFTWARE.
+ */
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+  <l:layout norefresh="true">
+    <l:main-panel>
+      <j:choose>
+        <j:when test="${h.isAnonymous()}">
+          <p>You are now logged out of Jenkins, however this has not logged you out of <a href="${it.gitHubURL}">GitHub</a>.</p>
```

Comment:
This should be corrected grammatically.  e.g.

> You are now logged out of Jenkins.  However, this has not logged you out of GitHub.

Also, it would be nice to have some conditional logic, like:

```xml
<p>You are now logged out of Jenkins, however this has not logged you out of <a href="${it.gitHubURL}">${if(it.githHubURL == 'https://github.com') { 'GitHub' } else { 'GitHub Enterprise' } }</a>.</p>
```